IHP/ModelSupport.hs

```diff
@@ -326,11 +326,16 @@ measureTimeIfLogging queryAction theQuery theParameters = do
     if currentLogLevel == Debug
         then do
             start <- getCurrentTime
-            result <- queryAction
-            end <- getCurrentTime
-            let theTime = end `diffUTCTime` start
-            logQuery theQuery theParameters theTime
-            pure result
+            result <- try $ queryAction
+
+            case result of
+                Left (e :: SomeException) -> do
+                    error $ "\nError in query: " <> show theQuery <> "\n" <> show e
```

Comment:
This will break some special error handling logic here: https://github.com/digitallyinduced/ihp/blob/master/IHP/ErrorController.hs#L172

We should add a `QueryException { query :: ByteString, pgException :: SomeException }` and then we could update the ErrorController to deal with that exception

Reply:
Just pushed this new exception onto this branch 👍